organizations/organization-logos.skos.ttl

```diff
@@ -0,0 +1,46 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix haOrg: <https://data.hetarchief.be/ns/organization/> .
+
+<https://data.hetarchief.be/id/logo-type/favicon> a skos:Concept ;
+    rdfs:subClassOf haOrg:Logo ;
+    skos:definition "Een klein pictogram van 24x24 pixels dat een website of webpagina vertegenwoordigt (in browsertabbladen, bladwijzers en soms in de adresbalk van een browser), waardoor gebruikers de site snel kunnen herkennen."@nl ;
```

Comment:
Maybe add English too? I also don't find a description of size 24x24 anywhere, most speak of 16x16 or 32x32.

Reply:
Apparently on AvO both 16x16 and 24x4 are used. I updated the definition to include both.